specification/databox/resource-manager/Microsoft.DataBox/stable/2019-09-01/databox.json

```diff
@@ -1318,6 +1711,73 @@
         }
       }
     },
+    "DataDestinationDetailsValidationRequest": {
+      "description": "Request to validate data destination details.",
+      "required": [
+        "destinationAccountDetails",
+        "location"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ValidationInputRequest"
+        }
+      ],
+      "properties": {
+        "destinationAccountDetails": {
+          "description": "Destination account details list.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DestinationAccountDetails"
+          }
+        },
+        "location": {
+          "description": "Location of stamp or geo.",
+          "type": "string"
+        }
+      },
+      "x-ms-discriminator-value": "ValidateDataDestinationDetails"
+    },
+    "DataDestinationDetailsValidationResponseProperties": {
+      "description": "Properties of data destination details validation response.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ValidationInputResponse"
+        }
+      ],
+      "properties": {
+        "status": {
+          "description": "Data destination details validation status.",
+          "enum": [
+            "Valid",
+            "Invalid",
+            "Skipped"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ValidationStatus",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "Valid",
+                "description": "Valid"
+              },
+              {
+                "value": "Invalid",
+                "description": "Invalid"
+              },
+              {
+                "value": "Skipped",
+                "description": "Skipped"
+              }
+            ]
+          }
+        }
+      },
+      "x-ms-discriminator-value": "ValidateDataDestinationDetails"
+    },
     "DestinationAccountDetails": {
       "description": "Details of the destination of the data",
       "required": [
```

Comment:
check with sanak if we need to deprecate account id in line 1815

Reply:
Checked with sanak. We need to deprecate it